src/status_im/contexts/wallet/common/scan_account/view.cljs

```diff
@@ -28,6 +28,7 @@
     [scan-qr-code/view
      {:title           (i18n/label :t/scan-qr)
       :subtitle        (i18n/label :t/scan-an-account-qr-code)
+      :share-button?   false
```

Comment:
I think this is redundant since if it is not passed its value would be `nil`

Also, I see we have 3 instances of `scan-qr-code/view` so we need to pass `:share-button?   true` to the ones where we need the share button (?)

Reply:
yeah, you're right. I'll swap the order of this and set the other ones to true and remove this 👍 
thanks!

Reply:
actually I think the other instances don't need it. Only the universal scanner does